Assets/Scripts/UI/InGame/Graphics/FPSLimiter.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FPSLimiter : MonoBehaviour {
+
+	
+	[SerializeField] public InputField fpsLimiter;
+	[SerializeField] public Dropdown fpsDropdown;
+	
+
+	public Dropdown FPSDropdown
+	{
+		get { return fpsDropdown; }
+		set { fpsDropdown = value; }
+	}
+	public InputField FPSLimit
+	{
+		get { return fpsLimiter; }
+		set { fpsLimiter = value; }
+	}
+	
+
+	private void Start() {
+		SetFPSLimit();
+	}
+
+	public void SetFPSLimit()
+	{
+		if(FPSDropdown.value == 5)
+		{
+			FPSLimit.interactable = true;
+			var text = FPSLimit.text;
+			if (FPSLimit.contentType.Equals(InputField.ContentType.IntegerNumber))
+			{
+				try
+				{
+					Application.targetFrameRate = int.Parse(text.ToString());
```

Comment:
Instead of using Parse and expecting an exception, please use int.TryParse https://docs.microsoft.com/en-us/dotnet/api/system.int32.tryparse?view=netframework-4.6